public/data/KendallDoesCoding.json

```diff
@@ -2,11 +2,22 @@
   "name": "Kendall Pinto",
   "type": "personal",
   "bio": "Hello, My name is Kendall. I am a Software Developer / Coder. I have made a bunch of programs in various languages, like Python, JavaScript, and the one I'm currently working on projects with, HTML & CSS. I'm currently working on the project, A Very Mogul Christmas",
-  "avatar": "https://i.ibb.co/5LsjwVB/Avatar-Maker.png",
+  "avatar": "https://github.com/KendallDoesCoding.png",
   "links": [
     {
       "name": "Check out my personal website",
-      "url": "https://kendalldoescoding.gq/",
+      "url": "https://kendalldoescoding.gq",
+      "icon": "globe"
+    },
+    {
+      "name": "Coding YouTube",
+      "url": "https://kendalldoescoding.gq/codingYT",
```

Comment:
Should, I just put the official youtube link as if my site goes down (even for some countries) this link wont work? My site only went down 2 times, inculding the one time I messed it up, so let me know if I should keep it as it is or change it @EddieHubCommunity/linkfree .

Reply:
It’s up to you and how long you intend to pay for the domain.